bin/ws_go

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [ "$#" -eq 0 ]; then
```

Comment:
Command line parsing should be improved - looks brittle to me. Also, should (optionally) allow -F as all other ws_* tools